base_tier_validation_server_action/tests/tier_validation_tester.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Ecosoft (http://ecosoft.co.th)
```

Comment:
I think you can use the same model generated on the test by `base_tier_validation`, so we can avoid this file

https://github.com/OCA/server-ux/blob/14.0/base_tier_validation/tests/tier_validation_tester.py

I don't know if it is possible

Reply:
It need one more field `test_bool` as _get_under_validation_exceptions() field. I am not sure we can inherit it further.

Or otherwise, we should have that field since `base_tier_validation`. WDYT?

Reply:
If you need that field, keep the file here. I didn't know if it was required :wink: